src/Features/ERDDAP/hooks/tabledap.ts

```diff
@@ -2,19 +2,24 @@
  * Hooks to acquire data from ERDDAP TableDAP datasets
  */
 import * as Sentry from "@sentry/react"
+import { useState } from "react"
 import { useQuery, QueryResult } from "react-query"
 
 import { resultToTimeseries, tabledapUrl } from "Shared/erddap"
 import { ErddapJson } from "Shared/erddap/types"
+import { groupBy } from "Shared/groupBy"
 import { proxytizeUrl } from "Shared/proxyUrl"
+import { aWeekAgoRounded } from "Shared/time"
 import { DataTimeSeries } from "Shared/timeSeries"
 
-import { PlatformTimeSeries } from "../types"
+import { PlatformTimeSeries, FetchGroup } from "../types"
 import { defaultQueryConfig } from "./hookConfig"
 
+const FORBIDDEN = "Forbidden"
+
 const getDataset = (timeSeries: PlatformTimeSeries, startTime?: Date) => {
   return async () => {
-    const url = urlBuilder(timeSeries, startTime)
+    const url = urlBuilder([timeSeries], startTime)
```

Comment:
Codacy found an issue: ['urlBuilder' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6227039)